packages/focuslock/test/setup.js

```diff
@@ -0,0 +1,4 @@
+import Adapter from 'enzyme-adapter-react-16'
```

Comment:
what would you think about migrating to react-testing-library?  Pros: We don't have to do version adapters.  We avoid the ginormous enzyme api.  We are forced more into blackbox testing and dom interrogation.